models/articles.model.js

```diff
@@ -11,10 +11,11 @@ exports.selectArticleById = (id) => {
     });
 };
 
-exports.selectArticles = () => {
-  return db
-    .query(
-      `SELECT
+exports.selectArticles = (topic) => {
+  const validFilterValues = ['mitch', 'cats', 'technology'];
```

Comment:
Hard coding these values is going to cause us some problems in the future when more topics are added to our api by our users. We will need to make use of an additional query to the database to determine whether the topic exists.